packages/cli-platform-apple/README.md

```diff
@@ -0,0 +1,46 @@
+# @react-native-community/cli-platform-apple
+
+This package is part of the [React Native CLI](../../README.md). It contains utilities for building reusable commands targetting Apple platforms.
+
+## Installation
+
+```sh
+yarn add @react-native-community/cli-platform-apple
+```
+
+## Usage
+
+This package is intended to be used internally in [React Native CLI](../../README.md) and by out of tree platforms.
+
+It exports builder commands that can be used to create custom `run-`, `log-` and `build-` commands for example: `yarn run-<oot-platform>`.
+
+Inside of `<oot-platform>/packages/react-native/react-native.config.js`:
+
+```js
+const {
+  buildOptions,
+  createBuild,
+} = require('@react-native-community/cli-platform-apple');
+
+const buildVisionOS = {
+  name: 'build-visionos',
+  description: 'builds your app for visionOS platform',
+  func: createBuild({platformName: 'visionos'}),
```

Comment:
nit: WDYT about simplifying this
```suggestion
  func: createBuild('visionos'),
```
and then add 
```
/**
*/
```
in function definition?

Reply:
Hmm, I wanted to keep this easily extendable and also for me without the "named parameter" it's unclear what this actually does at first glance.

Reply:
Alright, let's leave it as it is.